src/lcpubkey.cpp

```diff
@@ -538,6 +550,11 @@ bool PublicKey::verifyWithAppendix(std::istream& dataIn, const std::string& sign
     return _impl->verify<RSA_Appendix_Verifier, AppendixVerifyFilter>(dataWithSignature);
 }
 
+bool PublicKey::verifyWithPSSR_SHA1(std::istream& dataIn, std::ostream& dataOut) const
+{
+    return _impl->verify<RSA_PSSR_SHA1_Verifier, AppendixVerifyFilter>(dataIn, dataOut);
+}
+    
```

Comment:
See comment in `lcpubkey.cpp`

Reply:
My data already uses PSSR/SHA1 but maybe PSSR/SHA256 is more secure, so that it should be kept. Also, I would rather not have a deviation from your library in case that others (like you) contribute, extend it etc., so changing just the lib for my specific needs does not improve the library itself...

To keep your library backwards-compatible, one could also add template arguments that have default values. I personally do not see a great advantage in having fewer functions in lieu of more possible verifiers, but, of course, this is up to you.

Reply:
Alright.